### PR TITLE
[pt-br] Translation files from tsconfig-reference

### PR DESCRIPTION
**Pull request by oiagorodrigues**
First contribution. I've translated the following files from tsconfig section:

- [x] typeAcquisition.md

- [x] tsBuildInfoFile.md

- [x] traceResolution.md

- [x] target.md

- [x] suppressExcessPropertyErrors.md

- [x] suppressImplicitAnyIndexErrors.md

- [x] strict.md

- [x] strictBindCallApply.md

- [x] strictFunctionTypes.md

- [x] strictPropertyInitialization.md

- [x] stripInternal.md

@khaosdoctor, @alvarocamillont, @danilofuchs e @orta

### CODE REVIEW

File: packages/tsconfig-reference/copy/pt/options/strict.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
+display: "Strict"
```
Comment (khaosdoctor) [inline packages/tsconfig-reference/copy/pt/options/strict.md]:
```suggestion
display: "Modo estrito"
```

File: packages/tsconfig-reference/copy/pt/options/strictFunctionTypes.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+type StringOrNumberFunc = (ns: string | number) => void;
+
+// Atribuição não segura
+let func: StringOrNumberFunc = fn;
+// Chamada não segura - vai quebrar
+func(10);
+```
+
+Com `strictFunctionTypes` _on_, o erro é detectado corretamente:
```
Comment (khaosdoctor) [inline packages/tsconfig-reference/copy/pt/options/strictFunctionTypes.md]:
```suggestion
Com `strictFunctionTypes` ativado, o erro é detectado corretamente:
```

File: packages/tsconfig-reference/copy/pt/options/strictFunctionTypes.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+```ts twoslash
+// @errors: 2322
+function fn(x: string) {
+  console.log("Hello, " + x.toLowerCase());
```
Comment (khaosdoctor) [inline packages/tsconfig-reference/copy/pt/options/strictFunctionTypes.md]:
```suggestion
  console.log("Olá, " + x.toLowerCase());
```

File: packages/tsconfig-reference/copy/pt/options/strictFunctionTypes.md [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+}
+
+type StringOrNumberFunc = (ns: string | number) => void;
```
Comment (khaosdoctor) [inline packages/tsconfig-reference/copy/pt/options/strictFunctionTypes.md]:
```suggestion
type StringOuNumeroFn = (ns: string | number) => void;
```

File: packages/tsconfig-reference/copy/pt/options/strictFunctionTypes.md [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+
+// Atribuição não segura é prevenida
+let func: StringOrNumberFunc = fn;
```
Comment (khaosdoctor) [inline packages/tsconfig-reference/copy/pt/options/strictFunctionTypes.md]:
```suggestion
let func: StringOuNumeroFn = fn;
```

File: packages/tsconfig-reference/copy/pt/options/strictFunctionTypes.md [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+```
+
+Durante o desenvolvimento desse recurso, descobrimos um grande número de hierarquias de classes profundamente não seguras, incluindo algumas no DOM.
+Por causa disso, a configuração apenas se aplica a funções escritas na sintaxe _function_, não àquelas na sintaxe _method_:
+
+```ts twoslash
+type Methodish = {
```
Comment (khaosdoctor) [inline packages/tsconfig-reference/copy/pt/options/strictFunctionTypes.md]:
```suggestion
type PareceUmMetodo = {
```

File: packages/tsconfig-reference/copy/pt/options/strictFunctionTypes.md [hunk 7]
```diff
@@ -0,0 +1,55 @@
...
+  func(x: string | number): void;
+};
+
+function fn(x: string) {
+  console.log("Hello, " + x.toLowerCase());
```
Comment (khaosdoctor) [inline packages/tsconfig-reference/copy/pt/options/strictFunctionTypes.md]:
```suggestion
  console.log("Olá, " + x.toLowerCase());
```

File: packages/tsconfig-reference/copy/pt/options/strictFunctionTypes.md [hunk 8]
```diff
@@ -0,0 +1,55 @@
...
+}
+
+// Por fim, uma atribuição insegura, porém não detectada.
+const m: Methodish = {
```
Comment (khaosdoctor) [inline packages/tsconfig-reference/copy/pt/options/strictFunctionTypes.md]:
```suggestion
const m: PareceUmMetodo = {
```

File: packages/tsconfig-reference/copy/pt/options/strictPropertyInitialization.md [hunk 9]
```diff
@@ -0,0 +1,29 @@
+---
+display: "Inicialização de Propriedade Restrita"
```
Comment (khaosdoctor) [inline packages/tsconfig-reference/copy/pt/options/strictPropertyInitialization.md]:
```suggestion
display: "Inicialização restrita de propriedade"
```

File: packages/tsconfig-reference/copy/pt/options/strictPropertyInitialization.md [hunk 10]
```diff
@@ -0,0 +1,29 @@
...
+oneline: "Garante que todas as propriedades da classe correspondam aos seus tipos depois que o construtor finalizar"
+---
+
+Quando definido como verdadeiro, o TypeScript gerará um erro quando uma propriedade de classe for declarada, mas não definida no construtor.
```
Comment (khaosdoctor) [inline packages/tsconfig-reference/copy/pt/options/strictPropertyInitialization.md]:
```suggestion
Quando definido como `true`, o TypeScript gerará um erro quando uma propriedade de classe for declarada, mas não definida no construtor.
```

File: packages/tsconfig-reference/copy/pt/options/strictPropertyInitialization.md [hunk 11]
```diff
@@ -0,0 +1,29 @@
...
+  accountType = "user";
+
+  email: string;
+  address: string | undefined;
+
+  constructor(name: string) {
+    this.name = name;
+    // Note quer this.email não foi atribuído
+  }
+}
```
Comment (khaosdoctor) [inline packages/tsconfig-reference/copy/pt/options/strictPropertyInitialization.md]:
```suggestion
class Conta {
  nome: string;
  tipo = "usuario";

  email: string;
  endereco: string | undefined;

  constructor(nome: string) {
    this.nome = nome;
    // Note que this.email não foi atribuído
  }
}
```

File: packages/tsconfig-reference/copy/pt/options/strictPropertyInitialization.md [hunk 12]
```diff
@@ -0,0 +1,29 @@
...
+```
+
+No caso acima:
+
+- `this.name` é atribuído especificamente.
+- `this.accountType` é atribuído por padrão.
+- `this.email` não é atribuído e gera um erro.
+- `this.address` é declarado como possível `undefined`, o que significa que não precisa ser atribuído.
```
Comment (khaosdoctor) [inline packages/tsconfig-reference/copy/pt/options/strictPropertyInitialization.md]:
```suggestion
- `this.nome` é atribuído especificamente.
- `this.tipo` é atribuído por padrão.
- `this.email` não é atribuído e gera um erro.
- `this.endereco` é declarado como possível `undefined`, o que significa que não precisa ser atribuído.
```

File: packages/tsconfig-reference/copy/pt/options/stripInternal.md [hunk 13]
```diff
@@ -0,0 +1,60 @@
+---
+display: "Remover Internal"
```
Comment (khaosdoctor) [inline packages/tsconfig-reference/copy/pt/options/stripInternal.md]:
```suggestion
display: "Remover internal"
```

File: packages/tsconfig-reference/copy/pt/options/stripInternal.md [hunk 14]
```diff
@@ -0,0 +1,60 @@
...
+Não emite declarações para códigos que tenham uma anotação `@internal` em seu comentário JSDoc.
+Esta é uma opção interna do compilador; use por sua conta em risco, porque o compilador não verifica se o resultado é válido.
+Se você estiver procurando por uma ferramenta para lidar com níveis adicionais de visibilidade dentro de seus arquivos `d.ts`, veja o [api-extractor](https://api-extractor.com).
+
+```ts twoslash
+/**
+ * Dias disponíveis na semana
+ * @internal
+ */
+export const daysInAWeek = 7;
```
Comment (khaosdoctor) [inline packages/tsconfig-reference/copy/pt/options/stripInternal.md]:
```suggestion
export const diasNaSemana = 7;
```

File: packages/tsconfig-reference/copy/pt/options/stripInternal.md [hunk 15]
```diff
@@ -0,0 +1,60 @@
...
+
+/** Calcule quanto alguém ganha em uma semana */
+export function weeklySalary(dayRate: number) {
+  return daysInAWeek * dayRate;
```
Comment (khaosdoctor) [inline packages/tsconfig-reference/copy/pt/options/stripInternal.md]:
```suggestion
export function salarioSemanal(porDia: number) {
  return diasNaSemana * porDia;
```

File: packages/tsconfig-reference/copy/pt/options/stripInternal.md [hunk 16]
```diff
@@ -0,0 +1,60 @@
...
+}
```
Comment (khaosdoctor) [inline packages/tsconfig-reference/copy/pt/options/stripInternal.md]:
```suggestion
export const diasNaSemana = 7;

/** Calcule quanto alguém ganha em uma semana */
export function salarioSemanal(porDia: number) {
  return diasNaSemana * porDia;
}
```

File: packages/tsconfig-reference/copy/pt/options/stripInternal.md [hunk 17]
```diff
@@ -0,0 +1,60 @@
...
```
Comment (khaosdoctor) [inline packages/tsconfig-reference/copy/pt/options/stripInternal.md]:
```suggestion
export const diasNaSemana = 7;

/** Calcule quanto alguém ganha em uma semana */
export function selarioSemanal(porDia: number) {
  return diasNaSemana * porDia;
}
```

File: packages/tsconfig-reference/copy/pt/options/suppressExcessPropertyErrors.md [hunk 18]
```diff
@@ -0,0 +1,16 @@
+---
+display: "Prevenir Erros de Propriedades em Excesso"
```
Comment (khaosdoctor) [inline packages/tsconfig-reference/copy/pt/options/suppressExcessPropertyErrors.md]:
```suggestion
display: "Prevenir erros de propriedades em excesso"
```

File: packages/tsconfig-reference/copy/pt/options/suppressExcessPropertyErrors.md [hunk 19]
```diff
@@ -0,0 +1,16 @@
...
+oneline: "Permitir que propriedades adicionais sejam definidas durante a criação de tipos"
+---
+
+Desativa o relatório de erros de propriedades em excesso, como o mostrado no exemplo a seguir:
+
+```ts twoslash
+// @errors: 2322
+type Point = { x: number; y: number };
+const p: Point = { x: 1, y: 3, m: 10 };
```
Comment (khaosdoctor) [inline packages/tsconfig-reference/copy/pt/options/suppressExcessPropertyErrors.md:11]:
```suggestion
type Ponto = { x: number; y: number };
const p: Ponto = { x: 1, y: 3, m: 10 };
```

File: packages/tsconfig-reference/copy/pt/options/suppressImplicitAnyIndexErrors.md [hunk 20]
```diff
@@ -0,0 +1,25 @@
+---
+display: "Prevenir Erros de Implicit Any Index"
```
Comment (khaosdoctor) [inline packages/tsconfig-reference/copy/pt/options/suppressImplicitAnyIndexErrors.md]:
```suggestion
display: "Prevenir erros de implicit any index"
```

File: packages/tsconfig-reference/copy/pt/options/target.md [hunk 21]
```diff
@@ -0,0 +1,26 @@
+---
+display: "Target"
```
Comment (khaosdoctor) [inline packages/tsconfig-reference/copy/pt/options/target.md]:
```suggestion
display: "Runtime alvo"
```

File: packages/tsconfig-reference/copy/pt/options/traceResolution.md [hunk 22]
```diff
@@ -0,0 +1,9 @@
+---
+display: "Resolução de Rastreamento"
```
Comment (khaosdoctor) [inline packages/tsconfig-reference/copy/pt/options/traceResolution.md]:
```suggestion
display: "Resolução de rastreamento"
```

File: packages/tsconfig-reference/copy/pt/options/tsBuildInfoFile.md [hunk 23]
```diff
@@ -0,0 +1,10 @@
+---
+display: "Arquivo de Informações da Build do TS"
```
Comment (khaosdoctor) [inline packages/tsconfig-reference/copy/pt/options/tsBuildInfoFile.md]:
```suggestion
display: "Arquivo de informações da build do TS"
```

File: packages/tsconfig-reference/copy/pt/options/typeAcquisition.md [hunk 24]
```diff
@@ -0,0 +1,37 @@
+---
+display: "Aquisição de Tipo"
```
Comment (khaosdoctor) [inline packages/tsconfig-reference/copy/pt/options/typeAcquisition.md]:
```suggestion
display: "Aquisição de tipo"
```

File: packages/tsconfig-reference/copy/pt/options/typeAcquisition.md [hunk 25]
```diff
@@ -0,0 +1,37 @@
...
+oneline: "Conjunto de opções para Aquisição de Tipo Automática no JavaScript"
```
Comment (khaosdoctor) [inline packages/tsconfig-reference/copy/pt/options/typeAcquisition.md]:
```suggestion
oneline: "Conjunto de opções para aquisição de tipo automática no JavaScript"
```

**Review by khaosdoctor — CHANGES_REQUESTED** [review]

**Review by khaosdoctor — APPROVED** [review]

**Comment by khaosdoctor** [comment]
lgtm

**Comment by oiagorodrigues** [comment]
@khaosdoctor obrigado. Não pensei que você quisesse a tradução de todas as variavéis e classes. Vou manter isso em mente na próxima.

**Pull request merged**